File: Modul3/PRAK304-2310817110008-Muhammad_Raihan.py
a = int(input())
if a>=0 and a<100:
    if a>=1 and a<=9:
        print("Satuan")
    elif a>=11 and a<=19:
        print("Belasan")
    elif (a==10) or (a>=20 and a<=99):
        print("Puluhan")
    else:
        print("Nol")
elif a>=100:
    print("Anda Menginput Melebihi Limit Bilangan")

File: Modul3/PRAK304-2310817110008-Muhammad_Raihan.c
#include <stdio.h>

void main () {
    int a;
    scanf("%d", &a);
    if (a>=0 && a<100) {
        if (a>=1 && a<=9) {
            printf("Satuan");
        }
        else if (a>=11 && a<=19) {
            printf("Belasan");
        }
        else if ((a>=20 && a<=99) || (a==10)) {
            printf("Puluhan");
        }
        else {
            printf("Nol");
        }
    }
    else if (a>=100) {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
}